EasyPost.Tests/ServicesTests/WithParameters/Beta/UserServiceTest.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EasyPost.Models.API;
+using EasyPost.Models.API.Beta;
+using EasyPost.Tests._Utilities;
+using EasyPost.Tests._Utilities.Attributes;
+using EasyPost.Utilities.Internal.Attributes;
+using Xunit;
+
+namespace EasyPost.Tests.ServicesTests.WithParameters.Beta
+{
+    public class UserServiceTests : UnitTest
+    {
+        public UserServiceTests() : base("beta_user_service_with_parameters", TestUtils.ApiKey.Production) =>
```

Comment:
This isn't beta anymore right? It's also in the `Beta` directory?

```suggestion
        public UserServiceTests() : base("user_service_with_parameters", TestUtils.ApiKey.Production) =>
```

Reply:
Good catch, forgot to delete the file once I migrated stuff out of it.